docs/source/general/basics.rst

```diff
@@ -171,6 +171,8 @@ Template
 If you have devices with very similar *configuration dictionaries* you can store the shared
 blocks in one or more reusable templates which will be used as a base to build
 the final configuration.
+When using multiple templates, their order is important.
```

Comment:
I would add a new line before the sentence, plus a [note sign](http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions), like:

```
If you have devices with very similar *configuration dictionaries* you can store the shared blocks in one or more reusable templates which will be used as a base to build the final configuration.
 
.. note::
    when using multiple templates, their order is important.
    Templates that come afterwards overrides the templates that come before it.
```